packages/fuchsia_ctl/test/image_paver_test.dart

```diff
@@ -47,25 +60,27 @@ void main() {
   });
 
   test('Ssh fails', () async {
-    const FakeSshKeyManager sshKeyManager = FakeSshKeyManager(false);
-    final MemoryFileSystem fs = MemoryFileSystem(style: FileSystemStyle.posix);
-    final FakeTar tar = FakeTar(true, fs);
-    final MockProcessManager processManager = MockProcessManager();
-
+    sshKeyManager = const FakeSshKeyManager(false);
+    tar = FakeTar(true, fs);
     when(processManager.start(any)).thenAnswer((_) async {
       return FakeProcess(0, <String>['Good job'], <String>['']);
     });
 
     final ImagePaver paver = ImagePaver(
       tar: tar,
-      sshKeyManager: sshKeyManager,
+      sshKeyManagerProvider: (
+              {ProcessManager processManager,
+              FileSystem fs,
+              String pubKeyPath}) =>
+          sshKeyManager,
       fs: fs,
       processManager: processManager,
     );
 
     final OperationResult result = await paver.pave(
       'generic-x64.tgz',
       deviceName,
+      null,
```

Comment:
this should be a named parameter if it's sometimes not used.

Reply:
Done